CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Change Log
+
+All notable changes to this project will be documented in this file
+automatically by Versionist. DO NOT EDIT THIS FILE MANUALLY!
+This project adheres to [Semantic Versioning](http://semver.org/).
+
```

Comment:
Actually, looks like `versionist` handles this properly now and defaults to `1.0.0`, so this LGTM!